client/src/components/TrackList.vue

```diff
@@ -221,6 +221,8 @@ export default defineComponent({
       ];
     });
 
+    const virtualHeight = computed(() => props.height - 52);
```

Comment:
either prop or make note that this 52 is a magic number based on the `confidenceThresholdHeight` height located in `Sidebar.vue`?

Reply:
Great call.  See 73a80f -- I'm much happier with this now.  the `width` is now passed down as a prop and the max-width of the content area is a function of that width.  The CSS in `TypeItem` was simplified further to accept a runtime variable.